### PR TITLE
Fix: Set color for disabled days.

### PR DESCRIPTION
**Pull request by alextkachuk**
#### Changes
- Set disabled color for disabled days in the Calendar

#### Screenshots

capture

### CODE REVIEW

**Comment by OndrejCholeva** [comment]
LGTM, maybe consider disabled to be 1 level darker than outside-month so it is still easy to distinguish where the month starts and ends. @mcslash thoughts?

**Comment by omgaz** [comment]
LGTM

Are disabled days clickable?
Does the cursor change to cursor_win_hand

**Comment by alextkachuk** [comment]
@OndrejCholeva I thought about it, but Pedros said - it's ok.

@omgaz Disabled days are not clickable and have default cursor - it is original functionality, there was issue only with colour.

**Comment by patrick-ivers** [comment]
LGTM 🍶

**Comment by mcslash** [comment]
@alextkachuk as discussed, please make the border around the calendar part of the picker the same as the button border colour. ATM its too dark. Apart from that LGTM!

**Comment by omgaz** [comment]
Awesome! Rebase and fix Josh's comment and I'll merge

File: src/styles/_react-datepicker-custom.scss [hunk 1]
```diff
@@ -33,7 +33,7 @@
       border-radius: $datepicker-border-radius;
     }
 
-    &--outside-month {
+    &--outside-month, &--disabled {
```
Comment (OndrejCholeva) [inline src/styles/_react-datepicker-custom.scss]: should be on separate lines to make lint happy.

**Comment by alextkachuk** [comment]
capture

**Comment by alextkachuk** [comment]
@OndrejCholeva thanks - fixed

**Comment by OndrejCholeva** [comment]
LGTM :calendar:

**Comment by OndrejCholeva** [comment]
merging :rocket:

**Pull request merged**